tests/unit/module_utils/test_graphql_utils.py

```diff
@@ -15,7 +15,7 @@
 
 def test_setup_api_base():
     test_class = NautobotApiBase(
-        url="https://nautobot.example.com", token="abc123", validate_certs=False
+        url="https://nautobot.example.com", token="abc123", ssl_verify=False
     )
     assert isinstance(test_class.api, pynautobot.api)
     assert test_class.url == "https://nautobot.example.com"
```

Comment:
Do you want to add an assert to make sure `test_class.ssl_verify == True` and another for False?